Requirements.md

```diff
@@ -0,0 +1,31 @@
+# User Stories
+- As a cafe owner I want to be able to add menu items to a database for my customers to see.
+- As a cafe owner I would like to be able to reach out and advertise to my community effectively and on my own terms.
+- As a cafe owner I would like my customers to be able to see my menu online.
+- As a cafe owner I would like to be able to update menu items when prices or availability change.
+- As a manager I would like to offer discounts or incentives for app users.
+- As a cafe owner I would like for my customers to be able to pay online.
+- As an owner I would like to post pictures of my establishment and my menu items.
+- As a cafe owner I would like to be able to delete menu items seasonally or as supplies change.
+- As a cafe owner I would like to be able to offer my customers menu item modifications.
+
+# Feature Tasks
+[ ] CREATE batabase
```

Comment:
is this like the batcave? Quick robin, we must check the batData in the Batabase!